Add Windows zip document (#4045)
MicrosoftDocs/azure-docs-cli: docs-ref-conceptual/install-azure-cli-windows.md

The `-e` option is to ensure the official Azure CLI package is installed. This command installs the latest version by default. To specify a version, add a `-v <version>` with your desired version to the command.

# [ZIP Package](#tab/zip)

You can use the ZIP package to install the Azure CLI on Windows. This package is useful when you don't have administrative privilege.

> Please unzip the package to a folder, then you can run the Azure CLI with `<unzipped folder path>\bin\az.cmd`. (If the path contains spaces, please use `& "<unzipped folder path>\bin\az.cmd"` instead.)

Reviewer: @bebound, Is the above update true? I felt we needed to tell customers what terminals/environments this will work in.

> Please unzip the package to a folder, then you can run the Azure CLI with `<unzipped folder path>\bin\az.cmd`. (If the path contains spaces, please use `& "<unzipped folder path>\bin\az.cmd"` instead.)

Author: We don't support bash.
For PowerShell and CMD, we already mentioned it earlier _For Windows, the Azure CLI is installed via an MSI, which gives you access to the CLI through the Windows Command Prompt (CMD) or PowerShell._

You remind that `& "<unzipped folder path>\bin\az.cmd"` is only for PowerShell. I think it's tedious to add another command for CMD, and this is a common knowledge in Windows.

> Please unzip the package to a folder, then you can run the Azure CLI with `<unzipped folder path>\bin\az.cmd`. (If the path contains spaces, please use `& "<unzipped folder path>\bin\az.cmd"` instead.)

Reviewer 2: I suggest we do mention how to call it if the path contains space - in both CMD and PowerShell.

If you want to run the `az` command directly, then follow the steps below to add the Azure CLI to your PATH environment variable.
1. Open the Start Menu and search for `environment variables`, click `Edit the system environment variables`.
2. Click `Environment Variables...` button.
3. In the `User varibles for <username>` box, select `Path` and click `Edit...` button.
4. Click `New` button and add the `<unzipped folder path>\bin` to the list.
5. Restart your terminal and you should be able to run `az` command.

### Latest version
> [!div class="nextstepaction"]
> [Latest release of the Azure CLI (64-bit)](https://aka.ms/installazurecliwindowszipx64)

### Specific version
If you prefer, you can download a specific version of the Azure CLI by using a URL.

To download the ZIP package for a specific version, change the version segment in URL `https://azcliprod.blob.core.windows.net/zip/azure-cli-<version>-x64.zip`.

For example, to install the 64-bit ZIP of Azure CLI version 2.57.0, your URL would be `https://azcliprod.blob.core.windows.net/zip/azure-cli-2.57.0-x64.zip`. 

> Available Azure CLI versions can be found at [Azure CLI release notes](./release-notes-azure-cli.md). The ZIP package is available from version 2.57.0 and only 64-bit is available.

Reviewer: ```suggestion
Available Azure CLI versions can be found at [Azure CLI release notes](./release-notes-azure-cli.md). The ZIP package is available from version 2.57.0. Only 64-bit is available.
```

The `-e` option is to ensure the official Azure CLI package is installed. This command installs the latest version by default. To specify a version, add a `-v <version>` with your desired version to the command.

# [ZIP Package](#tab/zip)

> [!IMPORTANT]
> This package is currently in preview.

You can use the ZIP package to install the Azure CLI on Windows. This package is useful when you don't have administrative privilege.

> Please unzip the package to a folder, then you can run the Azure CLI with `<unzipped folder path>\bin\az.cmd`.

Reviewer: ```suggestion
Unzip the package to a folder, then using Cmd or PowerShell, install the Azure CLI by running `<unzipped folder path>\bin\az.cmd`.
```

> 2. Click `Environment Variables...` button.

Author: You can also use `settings -> system -> about ->Advanced system settings` to open this dialog, although it requires more steps.

If you want to run the `az` command directly, then follow the steps below to add the Azure CLI to your PATH environment variable.
1. Open the Start Menu and search for `environment variables`.  Click `Edit the system environment variables`.
1. Click the `Environment Variables...` button.
1. In the `User varibles for <username>` box, select `Path` and click the `Edit...` button.
1. Click the `New` button and add the `<unzipped folder path>\bin` to the list.
1. **Restart your terminal.**  You should now be able to run `az` commands.

### Latest version

> > [!div class="nextstepaction"]

Reviewer 2: Headings are usually followed by 2 line breaks.

> > [!div class="nextstepaction"]

Reviewer: ```suggestion

> [!div class="nextstepaction"]
```

The `-e` option is to ensure the official Azure CLI package is installed. This command installs the latest version by default. To specify a version, add a `-v <version>` with your desired version to the command.

# [ZIP Package](#tab/zip)

> [!IMPORTANT]
> This package is currently in preview.

You can use the ZIP package to install the Azure CLI on Windows. This package is useful when you don't have administrative privilege.

> Unzip the package to a folder, then using Cmd or PowerShell, call the Azure CLI by running `<unzipped folder path>\bin\az.cmd`.

Reviewer 2: Same as https://github.com/MicrosoftDocs/azure-docs-cli/pull/4045#discussion_r1473156429, I suggest we do mention how to call it if the path contains space - in both CMD and PowerShell.

Reviewer: I agree with @jiasli.  I don't feel it is possible to have too much clarity.  Is is possible to have too many words!  :-)

Our install docs receive the most negative feedback of any doc group in our repo, so I prefer not to wait for a customer complaint.

> If you want to run the `az` command directly, then follow the steps below to add the Azure CLI to your PATH environment variable.

Reviewer 2: ```suggestion
If you want to run the `az` command directly, follow the steps below to add the Azure CLI to your `PATH` environment variable.
```

1. Open the Start Menu and search for `environment variables`.  Click `Edit the system environment variables`.
1. Click the `Environment Variables...` button.
1. In the `User varibles for <username>` box, select `Path` and click the `Edit...` button.
1. Click the `New` button and add the `<unzipped folder path>\bin` to the list.
1. **Restart your terminal.**  You should now be able to run `az` commands.

### Latest version

> [!div class="nextstepaction"]

> > [Latest release of the Azure CLI (64-bit)](https://aka.ms/installazurecliwindowszipx64)

Reviewer 2: The button title will be the same as MSI:

![image](https://github.com/MicrosoftDocs/azure-docs-cli/assets/4003950/4ca7900f-7494-4479-a85e-c0dd264e3eb9)

How about using

- Latest MSI of the Azure CLI (32-bit)
- Latest MSI of the Azure CLI (64-bit)
- Latest ZIP of the Azure CLI (64-bit)

> ### Specific version

Reviewer 2: ```suggestion
### Specific version

```

> If you prefer, you can download a specific version of the Azure CLI by using a URL.

Reviewer 2: ```suggestion

If you prefer, you can download a specific version of the Azure CLI by using a URL.
```

> ### Specific version
> If you prefer, you can download a specific version of the Azure CLI by using a URL.

Reviewer 2: ```suggestion
### Specific version

If you prefer, you can download a specific version of the Azure CLI by using a URL.
```
